src/widgets/Warning.cpp

```diff
@@ -52,14 +53,14 @@ END_EVENT_TABLE()
 
 const TranslatableString &DefaultWarningFooter()
 {
-   static auto result = XXO("Don't show this warning again");
+   static auto result = XXO("Don't show this " + message1 + "again");
```

Comment:
avoid chopping up strings, they're a pain to translate. I believe this way would be better:
`wxString::Format( XXO("Don't show this %s again"), message1);`
see also https://github.com/audacity/audacity/wiki/Strings-&-translatable-code